crates/rolldown_plugin/src/plugin_driver/mod.rs

```diff
@@ -128,9 +134,10 @@ impl PluginDriver {
   }
 
   pub fn set_module_table(&self, module_table: &'static ModuleTable) {
-    self.contexts.iter().for_each(|ctx| {
-      ctx.module_table.set(module_table).expect("module_table is already set before");
-    });
+    if let Some(ctx) = self.contexts.first() {
+      let mut table = ctx.module_table.lock().unwrap();
+      *table = module_table;
```

Comment:
since the `module_table` are shared between all plugin, only write once is enough.